storm-core/src/jvm/backtype/storm/utils/MutableObject.java

```diff
@@ -18,7 +18,7 @@
 package backtype.storm.utils;
 
 public class MutableObject {
```

Comment:
@revans2 
I think we can remove this class. It is public class, but it's for internal usage, and can be replaced to AtomicReference as you changed.